README.md

```diff
@@ -1,3 +1,6 @@
 # GitHub Actions for IBM Power, IBM Z and LinuxONE Service Reference
 
 Please [submit an issue](https://github.com/IBM/actionspz/issues) on this repo to access GitHub Actions on Power and Z.
+
+## Endianness
+Note that IBM Power is little-endian and IBM Z is big-endian.
```

Comment:
IBM POWER (as in the CPU) would support both LE and BE. I would be great if BE variant GHA worker is also supported.